core/src/test/java/com/ugamsol/core/models/impl/BannerAreaImplTest.java

```diff
@@ -0,0 +1,57 @@
+package com.ugamsol.core.models.impl;
+
+import com.ugamsol.core.models.BannerArea;
+import io.wcm.testing.mock.aem.junit5.AemContext;
+import io.wcm.testing.mock.aem.junit5.AemContextExtension;
+import org.apache.sling.api.resource.Resource;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.*;
+@ExtendWith({AemContextExtension.class, MockitoExtension.class})
+
+class BannerAreaImplTest {
+
+    private final AemContext aemContext = new AemContext();
+    private BannerArea banner;
+
+    @BeforeEach
+    void setUp() {
+
+        //aemContext.addModelsForClasses(BannerAreaImpl.class);
```

Comment:
remove comments

Reply:
i have removed the comments and comitted again.